src/manifest.json

```diff
@@ -28,5 +28,17 @@
     "https://cmu-student-government.github.io/",
     "activeTab",
     "storage"
-  ]
+  ],
+
+  "applications": {
```

Comment:
what is this option responsible for? MDN doesn't define it:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json

Reply:
This is a browser specific settings option that currently only applies to Gecko.
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings

Reply:
Currently when the extension loads it assigns a randomly generated string to its ID, but this doesn't work too well if some browsers assume its ID is going to be the same across different sessions. This makes sure that it uses the same ID and can be referenced by other extensions by the same ID.

Reply:
Actually, strict_min_version might as well be 42a1 instead of 48... Nothing is stopping this extension from being useful in older versions of Firefox.